apps/web/src/pages/settings/organization/OrganizationName.styles.tsx

```diff
@@ -0,0 +1,9 @@
+import { css } from '../../../styled-system/css';
+
+export const InputWrapperStyles = css({
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
🤓 **nitpick (non-blocking):** We should avoid capitalizing styles like `ComponentNames` or `ClassNames`‏
```suggestion
export const inputWrapperStyles = css({
```

Reply:
The soul of Styled-components still lives in me 😅